ports/servoshell/browser.rs

```diff
@@ -545,6 +545,23 @@ where
                 EmbedderMsg::ReadyToPresent => {
                     need_present = true;
                 },
+                EmbedderMsg::EventDelivered(event) => match (browser_id, event) {
+                    (None, _) => {},
+                    (
+                        _,
+                        CompositorEventVariant::ResizeEvent |
+                        CompositorEventVariant::MouseMoveEvent |
+                        CompositorEventVariant::TouchEvent |
+                        CompositorEventVariant::WheelEvent |
+                        CompositorEventVariant::KeyboardEvent |
+                        CompositorEventVariant::CompositionEvent |
+                        CompositorEventVariant::IMEDismissedEvent,
+                    ) => {},
+                    (Some(browser_id), CompositorEventVariant::MouseButtonEvent) => {
+                        // TODO Focus browser and/or raise to top if needed.
+                        trace!("{}: Got a mouse button event", browser_id);
+                    },
```

Comment:
This will make this a bit shorter and easier to read, I think:

```suggestion
                    (Some(browser_id), CompositorEventVariant::MouseButtonEvent) => {
                        // TODO Focus browser and/or raise to top if needed.
                        trace!("{}: Got a mouse button event", browser_id);
                    },
                    (_, _) => {},
```

Reply:
done in 318c02ec839185d148db93cf3bc0b4ee4be9a37e